nodepool/nl01.sjc1.vexxhost.zuul.ansible.com.yaml

```diff
@@ -23,6 +23,10 @@ labels:
     max-ready-age: 600
   - name: esxi-6.7.0
     max-ready-age: 600
+  - name: esxi-6.7.0_with_nested
```

Comment:
what if we just called tihs
```
esxi-6.7.0-nested
````

Then left esxi-6.7.0 as the flavor?

But I think we should use - over _